test/features/user_can_click_home_and_clients_links_from_all_pages_test.rb

```diff
@@ -0,0 +1,90 @@
+require_relative "../test_helper"
+
+class UserCanClickTheHomeAndClientsLinksFromAnyPage < FeatureTest
+  include TestHelpers
+
+  def setup_tables_for_minimum_data_represented
+    @client = Client.create(identifier: "reddit", root_url: 'http://www.reddit.com')
+    populate_referrer_table
+    populate_agent_table
+    populate_url_paths_on_client #1=> /sports, 2 => /blog, 3 => /news
+    payload_1 = get_payload_data
+    payload_2 = get_payload_data
+    payload_2[:url_id] = 2
+    payload_3 = get_payload_data
+    payload_3[:url_id] = 3
+    create_payload([payload_1, payload_2, payload_3])
+  end
+
+  def test_user_can_click_the_home_link_from_any_page
```

Comment:
I really like the thoroughness of this and the next test. I also feel like they should probably be split into multiple tests since they are ~30 lines each.